style.css

```diff
@@ -0,0 +1,32 @@
+#title {
+  text-align: center;
+}
+
+#main {
+  background-color: lightgray;
+  margin: 30px auto;
+  max-width: 1280px;
+  max-height: 100%;
+  border-radius: 5px;
+  padding: 15px;
+  word-wrap: break-word;
+}
+
+#image{
```

Comment:
- [ ] Add a space before the opening { brackets to enable your linters to pass. It's a CSS syntax for clean code which makes it more readable for other developers working with your code.